run.sh

```diff
@@ -72,6 +72,10 @@ case "$deployment_action" in
         build_images
         deploy_osh
         ;;
+    "setup_airship")
+        setup_caasp_workers_for_openstack
```

Comment:
We should perhaps rename this role if it is not unique to openstack.

Reply:
Yes that will be good, may be as a part of overall reorganizing of osh and airship deploys